pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -7,6 +7,8 @@
 , backend            ? "webengine"
 , pipewireSupport    ? stdenv.isLinux
 , pipewire_0_2
+, enableWideVine     ? false
+, widevine-cdm
```

Comment:
Here's something to consider.

There are two attributes, `enableWideVine` and `widevine-cdm`, which could be replaced with:

`widevine-cdm ? null`

This makes it explicit that `widevine-cdm` is not used by default. The equivalent of `if enableWideVine then this else that` would be `if widevine-cdm != null then this else that`

Reply:
Are you sure it works this way? I tried to implement that but nix somehow always set the variable `widevine-cdm`. In order to *not* install widevine-cdm, I must override qutebrowser and set the `widevine-cdm` variable explicitely to `null` in my nixpkgs overlays (eg. `nixpkgs.overlays =  [ (final: prev: { qutebrowser = prev.qutebrowser.override { widevine-cdm = null; }; }) ];`).

Did I miss something?

[You can see my implementation here](https://github.com/jlamur/nixpkgs/commit/e3429b664a59e06134bb2b1a5c9e7d3bc4d486aa).

Reply:
You're right, my bad. Since `widevine-cdm` exists as a package, Nixpkgs will automatically provide it to the qutebrowser package. You'd have to update the qutebrowser definition in all-packages.nix to set `widevine-cdm` to null by default. It's probably not worth doing all of that.